### DISCUSSION TITLE
Alternative licensing

### DISCUSSION THREAD

**Discussion by zwass** [rhashimoto/wa-sqlite#74] [General]
Hi,

Your README suggests it might be possible to arrange a different than GPL license for this library. Can you provide more information on that? We are interested in using the library as a component of https://github.com/fleetdm/fleet, but our licensing model prohibits us from using GPLed libraries.

Thank you,
Zach

**Comment by rhashimoto** [comment]
Licensing inquiries can be e-mailed to licensing@shoestringresearch.com.
